bonedata_ws/src/air_data/launch/air_data_standalone.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+
+	<node pkg="air_data" name="air_data" type="air_data.py"
+	respawn="true" respawn_delay="3" output="screen"/>
+
+	<param name="svr_name" value="ubuntu" />
+	<param name="svr_ip" value="18.138.24.228" />
```

Comment:
The ip is mentioned in some files but not others, was this intentional?

Reply:
Yes. It is not mentioned in the air_data.launch file because that file obtains the param from air.launch